src/formInputs/DatePicker/index.tsx

```diff
@@ -51,6 +52,7 @@ export class DatePicker extends PureComponent<DatePickerProps, DatePickerState>
     useRange: false,
     rangeValue: {},
     alwaysShow: false,
+    showWeekEnd: false,
```

Comment:
```suggestion
    showWeekEnd: true,
```
로직에는 상관이 없고 뷰가 바뀌는 거라서 기본값을 true로 해도 될 것 같습니다. cc: @chiabi